db/config.hh

```diff
@@ -111,7 +111,6 @@ struct experimental_features_t {
         BROADCAST_TABLES,
         KEYSPACE_STORAGE_OPTIONS,
         TABLETS,
-        AUTH_V2,
```

Comment:
Please don't introduce stuff and remove it in the same series.

Reply:
ok, will re-squash parts of the commits, makes may live harder, but I guess worth it